Content.Client/Construction/UI/ConstructionMenu.xaml.cs

```diff
@@ -98,12 +104,12 @@ public ConstructionMenu()
             Recipes.OnItemDeselected += _ => RecipeSelected?.Invoke(this, null);
 
             SearchBar.OnTextChanged += _ =>
-                PopulateRecipes?.Invoke(this, (SearchBar.Text, Categories[OptionCategories.SelectedId]));
+                PopulateRecipes?.Invoke(this, (SearchBar.Text, Categories[OptionCategories.SelectedId] ,FuzzySearchEnabled));
```

Comment:
Comma here and on 112 is not properly placed (an IDE fixes this automatically for you)

Reply:
it's a stylistic choice

Reply:
Conforming to the style of the overall codebase is always better than making it inconsistent